.github/workflows/ci.yaml

```diff
@@ -64,9 +64,9 @@ jobs:
         MPLBACKEND: agg
 
     - name: Codecov
-      uses: codecov/codecov-action@v3
+      uses: codecov/codecov-action@v4
       with:
-        token: ${{ secrets.CODECOV_TOKEN }}
+        token: ${{ secrets.CODECOV }}
```

Comment:
Is this a change due to a merge or did you smuggle it into this PR??

Reply:
So you could see that the CI failed for codecov token issues, I tried to see if I could fix it.